### PR TITLE
[dedicated vc] management tool

### PR DESCRIPTION
**Pull request by mzmssg**
a part of #2790

### CODE REVIEW

File: src/tools/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.hadoop/
+*/.hadoop/
```
Comment (abuccts) [inline src/tools/.gitignore]: unnecessary, `.hadoop/` is enough.

File: src/tools/node_maintain.py [hunk 2]
```diff
@@ -236,6 +114,211 @@ def convert_nodes(nodes_str):
...
+    labeled_resources = get_resource_by_label(nodes_info)
+    for partition in labeled_resources:
+        if partition in dedicate_queues:
+            dedicate_queues[partition]["resource"] = labeled_resources[partition]["resource"]
+
+    for node in nodes_info:
+        if nodes_info[node]["nodeLabel"] in dedicate_queues:
+            dedicate_queues[nodes_info[node]["nodeLabel"]]["nodes"].append(node)
+    for queue_name, queue_attr in dedicate_queues.items():
+        print(queue_name + ":")
```
Comment (abuccts) [inline src/tools/node_maintain.py]: is this file py2 or py3? could you make it compatible?

File: src/tools/utility/log.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import os
+
+def setup_logging(default_path='config/logging.yaml', default_level=logging.INFO, env_key='LOG_CFG'):
+    path = default_path
+    value = os.getenv(env_key, None)
+    if value:
+        path = value
+    if os.path.exists(path):
+        with open(path, 'rt') as f:
+            config = yaml.load(f.read())
```
Comment (abuccts) [inline src/tools/utility/log.py]: why `f.read()` instead of `f`?

Comment (mzmssg) [reply src/tools/utility/log.py]: Done, use safe_load instead

File: src/tools/node_maintain.py (hunk 2 above)
Comment (mzmssg) [reply src/tools/node_maintain.py]: it's a dev-box script with python2, I made some compatible fix, but not fully tested, expecially unittest part.

File: src/tools/.gitignore (hunk 1 above)
Comment (mzmssg) [reply src/tools/.gitignore]: Done

File: src/tools/operator_wrapper/base_operator.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        stderr = subprocess.STDOUT if redirect_stderr else None
+        output = subprocess.check_output(command, stderr=stderr, shell=shell, **kwargs)
+        try:
+            output = output.decode("utf8")
+        except AttributeError:
+            pass
+        return output
+
+
+if __name__ == '__main__':
```
Comment (abuccts) [inline src/tools/operator_wrapper/base_operator.py]: Please unify `'` or `"`.

**Comment by mzmssg** [comment]
@abuccts 
Added lisence on non-doc parts.

**Pull request merged**